src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1172,6 +1172,15 @@ protected int GetAdjustedNodePosition(SyntaxNode node)
 
             var fullSpan = this.Root.FullSpan;
             var position = node.SpanStart;
+            if (node is StatementSyntax)
```

Comment:
Please either relax this condition (so it applies unconditionally), or file a new issue to relax this in milestone 2.1.